.evergreen/config.yml

```diff
@@ -387,12 +387,15 @@ buildvariants:
     tasks:
       - happy-eyeballs-task-group
 
-  - name: happy-eyeballs-windows
-    display_name: "Happy Eyeballs (Windows)"
-    run_on:
-      - windows-64-vs2017-small
-    tasks:
-      - happy-eyeballs-task-group
+#  Windows has a very short grace window before it starts refusing TCP handshakes, leading to flaky
+#  tests.
+#
+#  - name: happy-eyeballs-windows
+#    display_name: "Happy Eyeballs (Windows)"
+#    run_on:
+#      - windows-64-vs2017-small
+#    tasks:
+#      - happy-eyeballs-task-group
```

Comment:
I support removing testing on Windows, rather than commenting out. I do not expect it will be added back. There is still test coverage on macOS.

The one second delay may be expected behavior on Windows. This may be similar behavior noted in [PYTHON-356](https://jira.mongodb.org/browse/PYTHON-356):

> When a Windows client makes a socket connection to an unused port the Winsock stack will retry the failed connection 3 times, causing the connection failure to take over a second.